offix-offline/src/main/java/org/aerogear/offixoffline/Offix.kt

```diff
@@ -28,15 +33,138 @@ fun ApolloClient.enqueue(
     } else {
         Log.d("Extension", "Network not connected")
 
-        /* If the user is offline: (Case 1: When the app is in foreground, i.e. in-memory)
+        /* If the user is offline:
+           (For Case 1: When the app is in foreground, i.e. in-memory)
            1. Store the mutation object and callback in an array-list.
          */
         Log.d("Extension", " mutation : ${mutation.variables().valueMap()}")
         OfflineList.getInstance().offlineArrayList.add(mutation)
         OfflineList.getInstance().callbacksList.add(callback)
+
+        /* If the user is offline:
+          (For Case 2: When the app is in background, we will scheduleWorker a worker to replicate the mutations stored in database to the server)
+           1. Make an object of mutation persistence class.
+           2.Store it in database
+         */
+
+        /* Get access to the dao of the database
+         */
+        val libDao = getDao()
+
+        /* Insert mutation object in the database.
+        */
+        libDao?.insertMutation(getPersistenceMutation(mutation))
+        Log.d("Extension", " serial number: ${libDao?.getAllMutations()?.get(0)?.sNo}")
+        Log.d("Extension", " values of mutation: ${libDao?.getAllMutations()?.get(0)?.valueMap}")
+        Log.d("Extension", " size of db list after inserting mutations: ${libDao?.getAllMutations()?.size}")
+    }
+}
+
+/*
+ This function takes in an object of Mutation<D,T,V> and returns an object of Mutation persistence.
+ */
+fun getPersistenceMutation(mutation: Mutation<Operation.Data, Any, Operation.Variables>): org.aerogear.offixoffline.persistence.Mutation {
+
+    val operationId = mutation.operationId()
+    val operationDoc = mutation.queryDocument()
+    val operationName = mutation.name()
+    val valMap = mutation.variables().valueMap()
+    val jsonObject = JSONObject(valMap)
+    val responseClassName = mutation.javaClass.name
+
+    /* Make an object of Mutation persistence class
```

Comment:
Drop the `persistence` from keyword.
Or rename it to : 
```
Make an object of com.aerogear.offix.persistence.Mutation
```

Reply:
`org.aerogear.offixoffline.persistence.Mutation`

It's always a good idea to double-check the suggestions instead of directly copy-pasting them @Lavanyagaur22